assets/js/index.js

```diff
@@ -0,0 +1,110 @@
+
+document
+    .querySelector("ul")
+    .addEventListener("click", e =>{
+        if(e.target.tagName === 'LI'){
+            if(e.metaKey){
+                e.target.remove();
+            }
+            else{
+                e.target.style = 'text-decoration : line-through';
+            }
+        }
+    });
+
+for(let butt of document.querySelectorAll("button")){
+    butt.addEventListener("click", e =>{
+        for(let item of document.querySelectorAll("li")){
+            if(e.target.name === "mark_all"){
+                item.style = 'text-decoration : line-through';
+            }
+            else if(e.target.name === "delete_all"  && item.style.textDecoration === 'line-through'){
+                item.remove();
+            }
+        }
+    });
+}
+
+
+document.getElementById("write_item")
```

Comment:
Why not listen to form's `change` event and check if form is valid? This opens up the door for new "validation" bugs.